model-engine/model_engine_server/api/llms_v1.py

```diff
@@ -230,8 +232,14 @@ async def create_completion_stream_task(
         )
 
         async def event_generator():
-            async for message in response:
-                yield {"data": message.json()}
+            try:
+                async for message in response:
+                    yield {"data": message.json()}
+            except InvalidRequestException as exc:
+                yield {
```

Comment:
can you update docs about how to deal with errors when streaming

Reply:
yes, right now it looks like: 

```
stream = Completion.create(
    model="llama-2-7b",
    prompt="Give me a 200 word summary on the current economic events in the US.",
    max_new_tokens=1000,
    temperature=0.2,
    stream=True,
)

for response in stream:
  try:
      if response.output:
          print(response.output.text, end="")
          sys.stdout.flush()
  except: # an error occurred
    print(stream.text) # print the error message out 
```

Reply:
it sounds like you want to add them in a separate PR which is fine. be aware everything is in the same repo now

Reply:
I added them in this same pr